config/initializers/feature_toggles.rb

```diff
@@ -3,11 +3,7 @@
 beta_testers = ENV['BETA_TESTERS']&.split(/[\s,]+/) || []
 
 OpenFoodNetwork::FeatureToggle.enable(:customer_balance) do |user|
-  if beta_testers == ['all']
-    true
-  else
-    beta_testers.include?(user.email)
-  end
+  !Rails.env.test?
```

Comment:
This makes me a bit uncomfortable...

Reply:
Me too. It would have been good modify the specs in a way that they still pass. The simplest way would be to stub this method in related tests. That would make sure that all the other specs, which we think are not affected by this, are still passing and there is no surprising side effect.